### PR TITLE
Mhv 55412 copy mr to new location

### PR DESCRIPTION
**Pull request by mattwrightva**
## Summary
- src/applications/mhv/medical-records has been copied to src/applications/mhv-medical-records
- the manifest.json file was modified to reflect the testing state of the new app

## Related issue(s)
https://jira.devops.va.gov/browse/MHV-55412
> ### Description
> Isolated Apps - MR - Copy the Code to the new shared location

## Testing done
Local testing of the new Medical records app and all existing unit tests has been done.

## Screenshots
No UI changes

## What areas of the site does it impact?
- MHV Medical records new test app

## Acceptance criteria
Code has been copied to the new shared location on isolated apps

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

**Review by tynyttie — DISMISSED** [review]
Looks good.

**Review by tynyttie — DISMISSED** [review]

File: src/applications/mhv-medical-records/manifest.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "appName": "MHV Medical Records isolated app test",
+  "entryFile": "./app-entry.jsx",
+  "entryName": "medical-records-test",
+  "rootUrl": "/my-health/medical-records-test",
+  "productId": "c054e1e8-7be2-11ed-a1eb-0242ac120002-test"
```
Comment (mmoyer-va) [inline src/applications/mhv-medical-records/manifest.json:6]: This might be okay but all other apps use a UUID. Can we generate one for this?

**Review by mmoyer-va — CHANGES_REQUESTED** [review]

**Review by mmoyer-va — APPROVED** [review]
Looks good!

File: src/applications/mhv-medical-records/manifest.json (hunk 1 above)
Comment (mattwrightva) [reply src/applications/mhv-medical-records/manifest.json]: done!

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Pull request merged**